src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -104,7 +104,7 @@ export function getMatAutocompleteMissingPanelError(): Error {
     'aria-autocomplete': 'list',
     '[attr.aria-activedescendant]': 'activeOption?.id',
     '[attr.aria-expanded]': 'panelOpen.toString()',
-    '[attr.aria-owns]': 'autocomplete?.id',
+    '[attr.aria-owns]': '(autocomplete && autocomplete._isOpen) ? autocomplete.id : null',
```

Comment:
Can you move this logic into a getter method on the class and then use the attribute in the host binding?

Reply:
I remember having issues with NVDA not reading out some options when I tried doing this for `mat-select`. That being said, I couldn't hear any difference when I did it for autocomplete. Still it might be better to do this in a separate PR since this one is primarily for the tests.

Reply:
I think that we can still do it, but it should be done as a separate PR, because it needs a unit test and this PR is primarily focused on adding e2e tests to the autocomplete.